amethyst_core/src/system_ext.rs

```diff
@@ -10,6 +10,15 @@ use specs::prelude::{Read, System};
 pub trait SystemExt {
     /// Make a system pausable by tying it to a specific value of a resource.
     ///
+    /// When the value of the resource differs from `value` the system is considered "paused",
```

Comment:
Nice, nitpick incomming: "paused" => 'paused'

Reply:
Thanks, how come single quotes is correct here?

Reply:
Double quote is for marking a statement said by someone. If you just want to highlight a word you should use single quote.